cds_ils/patrons/serializers.py

```diff
@@ -47,15 +47,15 @@ def patron_loans_to_dict(patron_loans):
     :return: dict from patron's loan.
     :rtype: dict
     """
-    literatures_on_loan_results = patron_loans["active_loans"].hits.hits
+    literatures_on_loan_results = patron_loans["active_loans"].scan()
     literatures_on_loan = [
-        serialize_on_loan_literature_info(loan["_source"])
+        serialize_on_loan_literature_info(loan)
```

Comment:
just wondering if you were able to validate this change since the field changes...

Reply:
I tested it locally and it works correctly